Remove the mention of the two-month migration period (#152)
OlympusDAO-Education/Documentation: basics/migration.md

need to migrate if you want to spend more tokens (e.g. make new deposits) on these
platforms.

{% hint style="info" %}
Again, you would not miss any rebase rewards if you don't migrate during the migration

> phase. Even if you chose to migrate way later, you will receive the same amount

Reviewer: I meant this sentence "Even if you chose to migrate way later, you will receive the same amount
of gOHM that you would have received had you migrated immediately". Does this mean if you migrate later, you will not receive any rebases when migration? I don't think it's the case

Author: @Xosten no, the sentence is trying to tell that there is no difference between migrate later vs. migrate immediately

Author: @Xosten wdyt? if it is OK can you change this to approved?

Author: The conversation is resolved as the sentence is outdated

need to migrate if you want to spend more tokens (e.g. make new deposits) on these
platforms.

{% hint style="info" %}
You would not miss any rebase rewards if you don't migrate. Even if you chose to

> migrate after the migration phase, you will receive the same amount of gOHM that

Author: ```suggestion
migrate at a later time, you will receive the same amount of gOHM that
```

Author: Applied donmijoo's suggestion
